### DISCUSSION TITLE
tar_combine() some but not all branching variables?

### DISCUSSION THREAD

**Discussion by potash** [ropensci/tarchetypes#133] [General]
Another thing I am doing in my static branching pipelines is combining over some but not all of the variables in a tar_map(). E.g. if we mapped over variables `a` and `b` we then collapse to branching over `a` by combining all of the `b`s for a given value of `a`. I have some code to do this that I can share but I thought I'd check first whether there's already a way to do this. Thanks!

**Comment by wlandau** [comment]
I would have a look at `tar_select_names()` and `tar_select_targets()`. You can supply the output to `tar_combine()`.
